### DISCUSSION TITLE
Does GitHub hate Open University Student in Scotland?

### DISCUSSION THREAD

**Discussion by binarytango** [community/community#57079] [GitHub Education]
### Hi everyone,

I cannot get through the verification step because I am Open University Student in Scotland, and Github says I am not near a campus.

So how I am supposed to verify that I am a student?  Or does Github hate Open University students in Scotland?

**Comment by binarytango** [comment]
Third time lucky...

**Reply by samus-aran** [reply]
There are a number of factors taken into consideration. If you are a remote learner please ensure that your evidence indicates that you are a remote student - obviously this varies by institution. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this would've been a factor. 

But's it great you got verified and welcome to the GitHub Education community. We hope to see you around 👋
